cypress.json

```diff
@@ -7,5 +7,6 @@
     "retries": {
         "runMode": 2,
         "openMode": 0
-    }
+    },
+    "chromeWebSecurity": false
```

Comment:
The tests should also be runnable in firefox, do they work there?

Reply:
How does one run them in Firefox? (Should we be enabling that in CI?)

Reply:
![image](https://user-images.githubusercontent.com/1190097/173097481-7f427cae-5b9e-431d-8bdf-575b5308f9ac.png)

it even says it won't :D 

FF is the only one with the warning though:

![image](https://user-images.githubusercontent.com/1190097/173097590-158ae2dc-caf5-45a8-a1cd-a0def45d210d.png)

the tests also just don't run at all in FF:
![image](https://user-images.githubusercontent.com/1190097/173097750-22457462-6087-4c12-b354-0ee615092048.png)


bug: https://github.com/vector-im/element-web/issues/22524